rosdep/python.yaml

```
@@ -7347,6 +7347,16 @@ python3-pep8:
     pip:
       packages: [pep8]
   ubuntu: [python3-pep8]
+python3-pep8-naming-pip:
+  debian:
+    pip:
+      packages: [pep8-naming]
+  fedora:
+    pip:
+      packages: [pep8-naming]
+  ubuntu:
+    pip:
+      packages: [pep8-naming]
```

Actually, I take my approval back.  This package is available as a system package in a lot of places: see https://pkgs.org/download/python3-pep8-naming

```suggestion
python3-pep8-naming:
  debian: [python3-pep8-naming]
  fedora: [python3-pep8-naming]
  ubuntu: [python3-pep8-naming]
```

---

My bad. Fixed.